doc/changes/changes_2.0.0.md

```diff
@@ -29,6 +28,7 @@ The following new Scalar Functions are now supported:
 
 * #25: Refactored ScalarFunctionTests
 * #32: Removed `SQL_DIALECT` property
+* #17: Moved abstract scalar function test to virtual-schema-common-jdbc
```

Comment:
can we maybe order this list by ticket number?

```
* #17: Moved abstract scalar function test to virtual-schema-common-jdbc
* #25: Refactored ScalarFunctionTests
* #32: Removed `SQL_DIALECT` property
```

Reply:
it's ordered chronologically.